pure-docker/deploy-codeinsights-db.sh

```diff
@@ -27,6 +27,7 @@ docker run --detach \
     -e POSTGRES_USER=postgres \
     -e PGDATA=/var/lib/postgresql/data/pgdata \
     -v $VOLUME:/var/lib/postgresql/data/ \
+    -v $PWD/../codeinsights-db/conf:/conf \
```

Comment:
```suggestion
    -v $(pwd)/../codeinsights-db/conf:/conf \
```

Not sure it really matters but we use this syntax in deploy-grafana / deploy-prometheus. Full disclosure: I can't remember which approach is more universal.